source/_docs/scripts/conditions.markdown

```diff
@@ -206,7 +206,7 @@ The template condition tests if the [given template][template] renders a value e
 ```yaml
 condition:
   condition: template
-  value_template: "{% raw %}{{ (state_attr('device_tracker.iphone', 'battery_level')|int) > 50 }}{% endraw %}"
+  value_template: "{% raw %}'{{ (state_attr('device_tracker.iphone', 'battery_level')|int) > 50 }}'{% endraw %}"
```

Comment:
If you take away the `{% raw %]` tags, then this is what is left:

```
  value_template: "'{{ (state_attr('device_tracker.iphone', 'battery_level')|int) > 50 }}'"
```

Why does it need a quotation twice?

Reply:
PS: it helps if you used the PR template that was provided when you opened up the PR. Since it will ask for a description, which is currently now missing from your PR. Could you elaborate?